OsmIntegrator/Controllers/AccountController.cs

```diff
@@ -88,6 +88,7 @@ public async Task<ActionResult<TokenData>> Login([FromBody] LoginData model)
 
       if (userEmail == null)
       {
+        // TODO: security vulnerability - disclosing users data
```

Comment:
Good point.
What should be the message?
The same as below? `The username or password were not correct. Try again`?

Reply:
New bug has been created: https://rozwiazaniadlaniewidomych.atlassian.net/browse/PT-335

Reply:
Yes, that should suffice